executable/microsoft_pe.ksy

```diff
@@ -408,3 +413,144 @@ types:
         size: length - 8
         doc: Contains a certificate, such as an Authenticode signature.
     doc-ref: 'https://docs.microsoft.com/en-us/windows/desktop/debug/pe-format#the-attribute-certificate-table-image-only'
+
+  resource_directory_table:
+    params:
+      - id: depth
+        type: u4
+      - id: rsrc_section_file_offset
+        type: u4
+      - id: rsrc_section_virtual_address
+        type: u4
+    seq:
+      - id: characteristics
+        type: u4
+      - id: time_date_stamp
+        type: u4
+      - id: major_version
+        type: u2
+      - id: minor_version
+        type: u2
+      - id: number_of_named_entries
```

Comment:
code style prescribes `num_` prefix, but IMHO the word `number` is too generic for the purpose, in own specs I use `count`.

Reply:
I agree with your statement although I think it's preferable to stick to the style guide. I've used `num_` instead of `number_of_`.